lib/vibrio/src/rumprt/crt/mod.rs

```diff
@@ -248,36 +252,46 @@ pub extern "C" fn main() {
 
     unsafe {
         log::set_logger(&crate::writer::LOGGER)
-            .map(|()| log::set_max_level(Level::Error.to_level_filter()))
+            .map(|()| log::set_max_level(Level::Debug.to_level_filter()))
```

Comment:
just fyi this might slow-down benchmarks by accident (too much logging because we log directly to serial and need to acq. a lock for it)